core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -293,6 +293,8 @@
       "The maximum number of concurrent tablet migrations for a tablet server"),
   TSERV_MAJC_MAXCONCURRENT("tserver.compaction.major.concurrent.max", "3", PropertyType.COUNT,
       "The maximum number of concurrent major compactions for a tablet server"),
+  TSERV_MAJC_THROUGHPUT("tserver.compaction.major.throughput", "0B", PropertyType.MEMORY,
+      "Maximum number of bytes to read or write per second over all major compactions, or 0 for unlimited."),
```

Comment:
".... in a TabletServer." Clarify that this is not global, but local to a tserver.